### PR TITLE
[Task#48429] Setup databse, pusher, laravel echo, pusher js

### PR DESCRIPTION
**Pull request by khieutrung12**
## Related Tickets:

-   [#48429](https://edu-redmine.sun-asterisk.vn/issues/48429)

## WHAT this PR does?

- install pusher, laravel echo, pusher js
- database Notifications
- 
## Run
- composer install
- npm install
- bower install
- npm run dev
## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

**Comment by khieutrung12** [comment]
ready

File: resources/js/notification_order_status.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+$(document).ready(function() {
+      if(userID) {
+          window.Echo.private(`order.${userID}`)
+          .notification((notification) => {
+                showNotifications(notification, '#notifications');
+                var notificationsWrapper = $('.dropdown-notifications');
+                var notificationsToggle = notificationsWrapper.find('a[data-toggle]');
+                var notificationsCountElem = notificationsToggle.find('i[data-count]');
+                var notificationsCount = parseInt(notificationsCountElem.data('count'));
+                console.log(notificationsCount);
```
Comment (huyenttm-0901) [inline resources/js/notification_order_status.js]: bỏ console đi em

**Comment by khieutrung12** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**